go/ai/gen.go

```diff
@@ -203,3 +203,13 @@ type ToolResponse struct {
 	// An example might be map[string]any{"name":"Thomas Jefferson", "born":1743}.
 	Output map[string]any `json:"output,omitempty"`
 }
+
+// NewTextMessages is a helper function that creates a slice of [Message] from one role and string.
+func NewTextMessages(r Role, m string) []*Message {
```

Comment:
wee can iterate on this, but "New Text Messages" from single string reads a bit weird...

Reply:
I agree, I'm open to ideas because I haven't been able to come up with anything better. :-)

Reply:
Alternatively: `NewTextMessageAsSlice()`

Reply:
It's fine, we can iterate later.